### ISSUE TITLE
As a user I want to know which methods are available to me to purchase my permit at pay.gov when I select the "Buy Permit" button

### ISSUE THREAD

**Issue by ASprinkle** [nciinc/fs-permit-platform#398]
This could require a text change on the "Buy permit with credit card" button to be "Buy Permit" and then specify the pay.gov methods on the buy permit page.

**Comment by sadlerw** [comment]
Simplest solution is just to remove the "with credit card" and allow the user to see the options once they click the button to go to pay.gov  Then when they don't take Discover they can abandon the page with a huff. Other ideas @JaneZC @Npage123 @pvd-flexion                     

![Screen Shot 2018-02-09 at 8.37.28 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/f12bd976-5663-422e-bcbc-2252aecc05bf)

**Comment by JaneZC** [comment]
I'm all for simple!

**Comment by npage123** [comment]
I agree as well. Is there contact info when they get to pay.gov on if their wanted payment option isn't on there like Discover?

**Comment by sadlerw** [comment]
it's either pay with these available options or go to your local forest office - maybe we need a story to make sure we have that somewhere? @JaneZC do we state that permits can be purchased in person at any district office someplace?

**Comment by ASprinkle** [comment]
@JaneZC @sadlerw It is on the landing page, but kind of hidden "For national forests that do not offer the online permit option, you can buy a permit at the local Ranger District office." I don't see that disclaimer anywhere else about how permits can also be purchased at offices (maybe it should be repeated under contact us section). I will make a story for that in new issues.

**Comment by sadlerw** [comment]
Maybe something like this?

![Screen Shot 2018-02-09 at 12.01.44 PM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/a71ea355-f3b0-425a-bd03-0ce7c5e6fcea)

**Comment by MelissaBraxton** [comment]
A few minor tweaks: 
- Move the text to below the checkout button so that it's not grouped with the agreement and info exclusive to the online transaction.  We also don't want it to appear as though "checkout" will let users purchase a permit from a District office. 
- Make it more clear that purchasing a permit at a district office cannot be done online and that it requires visiting an office. E.g., "You can also buy a permit offline by visiting a Ranger District office."

**Comment by sethalt** [comment]
@MelissaBraxton How about this?

![screen shot 2018-02-14 at 10 40 01 am](https://user-images.githubusercontent.com/2347759/36216150-8ffd4f6e-1173-11e8-99bd-95579c1a51d5.png)

**Comment by npage123** [comment]
@sethalt @MelissaBraxton - I'm thinking we still want to notify them of pay.gov disclaimer before the button? And just the "You can also buy a permit offline by visiting a Ranger District office." below the checkout button.

**Comment by sethalt** [comment]
How's this @npage123 @MelissaBraxton @ASprinkle? 
I think using the hint text style on pay.gov disclaimer differentiates enough from the terms agreement bit.

![screen shot 2018-02-14 at 11 13 18 am](https://user-images.githubusercontent.com/2347759/36217845-162c6d00-1178-11e8-9203-f89341c42d3a.png)

**Comment by MelissaBraxton** [comment]
Looks good to me! 👍
